extensions/slack/src/shared/client/WebClient.ts

```diff
@@ -2,6 +2,11 @@ import { getPreferenceValues } from "@raycast/api";
 import { OAuthService } from "@raycast/utils";
 import { WebClient } from "@slack/web-api";
 
+export type Preferences = {
+  timeFormat: "hour12" | "hour24";
+  displayExtraMetadata: boolean;
+};
```

Comment:
Nitpick (non-blocking): this is no longer needed and you can just use `Preferences` created in _raycast-env.d.ts_: [src](https://developers.raycast.com/api-reference/preferences#getpreferencevalues)

You will need to add this to tsconfig.json though:
https://github.com/raycast/extensions/blob/adda091532b4805defd86396fc2b6ddb2eae05f3/extensions/todoist/tsconfig.json#L4